### ISSUE TITLE
another "how did I get here" panic

### ISSUE THREAD

**Issue by SvenDowideit** [chrislusf/seaweedfs#1814]
I'm basically just making changes to the Docker volume driver - and managed to make this happen - sorry.

```
root@onaci-001:/var/lib/docker/plugins/swarm/rootfs/mnt/docker-volumes/grafana.tower-sl.ona.im_config# docker logs -f seaweed-volume-plugin-grafana.tower-sl.ona.im_config
I0218 05:58:25     1 config.go:29] Reading security.toml from 
I0218 05:58:25     1 config.go:33] Reading : Config File "security" Not Found in "[/ /root/.seaweedfs /usr/local/etc/seaweedfs /etc/seaweedfs]"
This is SeaweedFS version 30GB 2.24 a833021 linux amd64
mount point owner uid=0 gid=0 mode=drwxr-xr-x
current uid=0 gid=0
I0218 05:58:25     1 volume_create_linux.go:19] Preallocated 65536000 bytes disk space for /tmp/5ec20dac/c0_2_0.dat
I0218 05:58:25     1 chunk_cache_on_disk.go:66] loading leveldb /tmp/5ec20dac/c0_2_0.ldb
I0218 05:58:25     1 needle_map_leveldb.go:30] Start to Generate /tmp/5ec20dac/c0_2_0.ldb from /tmp/5ec20dac/c0_2_0.idx
I0218 05:58:25     1 needle_map_leveldb.go:32] Finished Generating /tmp/5ec20dac/c0_2_0.ldb from /tmp/5ec20dac/c0_2_0.idx

------- >8 snip -----------------------------------

I0218 05:58:25     1 mount_std.go:210] mounted filer:8888/mnt/docker-volumes/grafana.tower-sl.ona.im_config to /mnt/docker-volumes/grafana.tower-sl.ona.im_config/_data
2021/02/18 05:58:27 fuse: panic in handler for Getxattr [ID=0x2 Node=0x1 Uid=0 Gid=0 Pid=0] "security.capability" 24 @0: runtime error: slice bounds out of range [50:19]
goroutine 146 [running]:
github.com/seaweedfs/fuse/fs.(*Server).serve.func2(0x266ca60, 0xc00015ec30, 0xc00073ae78, 0xc00073adee)
	/home/travis/gopath/pkg/mod/github.com/seaweedfs/fuse@v1.1.1/fs/serve.go:937 +0x1af
panic(0x20fcb20, 0xc00084c3c0)
	/home/travis/.gimme/versions/go1.15.8.linux.amd64/src/runtime/panic.go:969 +0x1b9
github.com/chrislusf/seaweedfs/weed/util/bounded_tree.(*BoundedTree).EnsureVisited(0xc0004be660, 0xc0003c3800, 0x13, 0xc00073a9c0, 0x0, 0x0)
	/home/travis/gopath/src/github.com/chrislusf/seaweedfs/weed/util/bounded_tree/bounded_tree.go:45 +0x230
github.com/chrislusf/seaweedfs/weed/filesys/meta_cache.EnsureVisited(0xc00083f780, 0x265b760, 0xc000838200, 0xc0003c3800, 0x13, 0xc0003c3814, 0x1e)
	/home/travis/gopath/src/github.com/chrislusf/seaweedfs/weed/filesys/meta_cache/meta_cache_init.go:15 +0x9b
github.com/chrislusf/seaweedfs/weed/filesys.(*WFS).maybeLoadEntry(0xc000838200, 0xc0003c3800, 0x13, 0xc0003c3814, 0x1e, 0x1e, 0x1, 0xc0001ae420)
	/home/travis/gopath/src/github.com/chrislusf/seaweedfs/weed/filesys/xattr.go:117 +0x106
github.com/chrislusf/seaweedfs/weed/filesys.(*Dir).maybeLoadEntry(0xc000645ec0, 0x32, 0xc0001ae420)
	/home/travis/gopath/src/github.com/chrislusf/seaweedfs/weed/filesys/dir.go:515 +0x9f
github.com/chrislusf/seaweedfs/weed/filesys.(*Dir).Getxattr(0xc000645ec0, 0x267cda0, 0xc00083f900, 0xc00015ec30, 0xc00000d9e0, 0x1, 0x0)
	/home/travis/gopath/src/github.com/chrislusf/seaweedfs/weed/filesys/dir.go:77 +0x96
github.com/seaweedfs/fuse/fs.(*Server).handleRequest(0xc000162180, 0x267cda0, 0xc00083f900, 0x263c120, 0xc000645ec0, 0xc0007e88c0, 0x266ca60, 0xc00015ec30, 0xc00073ae78, 0xc0006414a0, ...)
	/home/travis/gopath/pkg/mod/github.com/seaweedfs/fuse@v1.1.1/fs/serve.go:1207 +0x235b
github.com/seaweedfs/fuse/fs.(*Server).serve(0xc000162180, 0x266ca60, 0xc00015ec30)
	/home/travis/gopath/pkg/mod/github.com/seaweedfs/fuse@v1.1.1/fs/serve.go:958 +0x2c7
github.com/seaweedfs/fuse/fs.(*Server).Serve.func1(0x0)
	/home/travis/gopath/pkg/mod/github.com/seaweedfs/fuse@v1.1.1/fs/serve.go:472 +0x79
github.com/seaweedfs/fuse/fs.(*Server).Serve.func2(0xc000162180, 0xc00021d800, 0x0)
	/home/travis/gopath/pkg/mod/github.com/seaweedfs/fuse@v1.1.1/fs/serve.go:479 +0x64
created by github.com/seaweedfs/fuse/fs.(*Server).Serve
	/home/travis/gopath/pkg/mod/github.com/seaweedfs/fuse@v1.1.1/fs/serve.go:477 +0x3b0

```

**Comment by SvenDowideit** [comment]
possibly worse, is that weed doesn't exit, the container hangs around, and so things can't retry

**Comment by chrislusf** [comment]
this duplicates with https://github.com/chrislusf/seaweedfs/issues/1799
it has been fixed in 2.26

**Comment by SvenDowideit** [comment]
blooming awesome! :D
